public/index.html

```diff
@@ -23,6 +23,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Met Data Portal</title>
+    <script src="%PUBLIC_URL%/env.js"></script>
```

Comment:
A little comment here would help clarify what happens if dev and prod environments.